### PR TITLE
Completed assignment 2 

### PR DESCRIPTION
**Pull request by VictorNS69**
This PR will also merge the #2 .

Please advise me if there is any conflict.

### CODE REVIEW

File: Assignment2/VictorNS69-Victor Nieves.rdf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    </rdf:Description>	
+    <rdf:Description rdf:about="#Computer101">
+		<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+		<hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401">
+		<hasTemperature>29</hasTemperature>
+		<atTime>2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.rdf]: It is good practice to datatype these literals.

File: Assignment2/VictorNS69-Victor Nieves.ttl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+:Class01 includes :Sensor029;
+	includes :Computer101.
+:Computer101 hasOwner :User10A.
+:User10A hasName :"Pedro".
+
+:Sensor029 hasMeasurement :Measurement8401.
+:Measurement8401 hasTemperature :"29";
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.ttl]: Literals should be datatyped, and in fact the syntax here is wrong.

File: Assignment2/VictorNS69-Victor Nieves.ttl [hunk 3]
```diff
@@ -0,0 +1,8 @@
+:Class01 includes :Sensor029;
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.ttl]: Prefixes missing

File: Assignment2/VictorNS69-Victor Nieves.rdf (hunk 1 above)
Comment (VictorNS69) [reply Assignment2/VictorNS69-Victor Nieves.rdf]: Is this now ok?

https://github.com/VictorNS69/Curso2019-2020/blob/e54b85b3442dd1f857e415610e8d2206f6fff9e1/Assignment2/VictorNS69-Victor%20Nieves.rdf#L16

File: Assignment2/VictorNS69-Victor Nieves.ttl (hunk 2 above)
Comment (VictorNS69) [reply Assignment2/VictorNS69-Victor Nieves.ttl]: Changed in the last commit

File: Assignment2/VictorNS69-Victor Nieves.ttl (hunk 3 above)
Comment (VictorNS69) [reply Assignment2/VictorNS69-Victor Nieves.ttl]: Changed in the last commit

**Comment by VictorNS69** [comment]
I fixed the conflicts manually. You can merge now.

File: Assignment2/VictorNS69-Victor Nieves.rdf [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <rdf:Description rdf:about="#Class01">
+		<includes rdf:resource="#Sensor029"/>
+		<includes rdf:resource="#Computer101"/>
+    </rdf:Description>	
+    <rdf:Description rdf:about="#Computer101">
+		<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+		<hasMeasurement rdf:resource="#Measurement8401"/>
+	    	<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.rdf]: Not needed, but ok

File: Assignment2/VictorNS69-Victor Nieves.ttl [hunk 5]
```diff
@@ -0,0 +1,11 @@
+@prefix example: <http://www.ontogrid.net/Assignment2/1.b>
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+:Class01 example:includes :Sensor029;
+	example:includes :Computer101.
+
+:Computer101 example:hasOwner :User10A.
+:User10A example:hasName :"Pedro".
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.ttl]: No : before "Pedro"

Comment (VictorNS69) [reply Assignment2/VictorNS69-Victor Nieves.ttl]: Fixed

File: Assignment2/VictorNS69-Victor Nieves.ttl [hunk 6]
```diff
@@ -0,0 +1,11 @@
+@prefix example: <http://www.ontogrid.net/Assignment2/1.b>
```
Comment (ocorcho) [inline Assignment2/VictorNS69-Victor Nieves.ttl]: You should add a base prefix, but ok

**Pull request merged**